src/components/schemas/employee.yaml

```diff
@@ -586,3 +586,31 @@ employees_download_out:
         - PROCESSING
       description: |
         State of this download.
+
+
+spender_commute_details_in:
```

Comment:
why is this different ?

Reply:
didn't understand this, you mean why declare it separately?
- if yes, then that's because in input payload, we only accept `commute_details` key
- but in response, along with `commute_details`, we are also sending `user_id`, `user_email` etc.